pkgs/top-level/python-packages.nix

```diff
@@ -28490,15 +28490,16 @@ in modules // {
 
   mps-youtube = buildPythonPackage rec {
     name = "mps-youtube-${version}";
-    version = "0.2.6";
+    version = "0.2.7.1";
```

Comment:
I have changed this pull request that the content of the request fits to the title. Please reopen the pull request #18636.